### ISSUE TITLE
415 Unsupported Media Type while trying to use getInvoice

### ISSUE THREAD

**Issue by alozytskyi** [square/square-php-sdk#105]
**Describe the bug**
While calling `getInvoice` using latest SDK, I get 415 unsupported media type. This request works fine in API explorer.

**Expected behavior**
Invoice must be returned normally

**To Reproduce**
Steps to reproduce the bug:
1. Create an invoice in sandbox
2. Try get it with `$client->getInvoicesApi()->getInvoice('inv:...')`
3. Get 415 response code 

**Screenshots**
From sandbox API log:
![image](https://user-images.githubusercontent.com/25198154/217567650-362c10d0-4ecb-44c3-9bdc-01028cefe4f9.png)

Request:
```
content-type: text/plain; charset=utf-8
square-version: 2023-01-19
accept-encoding: deflate, gzip
accept: application/json
user-agent: Square-PHP-SDK/25.1.0.20230119 (2023-01-19) Zend/4.1.12 (omitted due to privacy reasons)
```

Response: 
```
date: Wed, 08 Feb 2023 14:46:07 GMT
transfer-encoding: chunked
vary: Accept
x-frame-options: SAMEORIGIN
connection: keep-alive
content-type: text/plain; charset=utf-8
cache-control: no-cache
```

**Square SDK version**
25.1.0.20230119

**Additional context**

**Comment by zenmasterjobo** [comment]
hi @alozytskyi,

Looks like we have an issue with the sdk that we need to address, as well as with some of our APIs.

In the mean time you can manually change the code in the SDK to set an `application/json` content-type onto the request header.

Inside of `vendor/square/square/src/Apis/InvoicesAPI.php`

In the `getInvoice()` method you can add the `'Content-Type': 'application/json'` like so 
```
->parameters(TemplateParam::init('invoice_id', $invoiceId),
                         HeaderParam::init('Content-Type', 'application/json'));
```

Let me know if you have any further questions!

**Comment by zenmasterjobo** [comment]
Hi @alozytskyi ,

I just wanted to confirm that this has been fixed as of version [32.0.0.20231018](https://github.com/square/square-php-sdk/releases/tag/32.0.0.20231018) of the php sdk. Please feel free to reopen this issue if you run into further problems!
